src/components/PlayerView/playerViewConfig.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Configures the visual presentation and behaviour of the `PlayerView`.
+ */
+export interface PlayerViewConfig {
+  /**
+   * Configures the visual presentation and behaviour of the Bitmovin Player UI.
+   * A `WebUiConfig` can be used to configure the default Bitmovin Player Web UI.
+   *
+   * Default is `WebUiConfig`.
+   *
+   * Limitations:
+   * Configuring the `uiConfig` only has effect if the `StyleConfig.userInterfaceType` is set to `Bitmovin`.
+   */
```

Comment:
This way it would link all those types on the API reference properly with a hyperlink.

```suggestion
  /**
   * Configures the visual presentation and behaviour of the Bitmovin Player UI.
   * A {@link WebUiConfig} can be used to configure the default Bitmovin Player Web UI.
   *
   * Default is {@link WebUiConfig}.
   *
   * Limitations:
   * Configuring the `uiConfig` only has effect if the {@link StyleConfig.userInterfaceType} is set to {@link UserInterfaceType.Bitmovin|Bitmovin}.
   */
```